### PR TITLE
Salesperson is_active flag.

### LINKED ISSUE #46: Replace salesperson delete action with is_active flag
Using soft deletes to disable a salesperson is counter intuitive and possibly confusing.

Change this behavior so that delete is no longer available and a salesperson (or other entity trait information) can only be made inactive (is_active = 0).

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Control salesperson active state with is_active flag rather than soft delete.
Replace Salesperson PrimaryLocation with a belongs_to Location.
 - Closes #46 and #54.

### CODE REVIEW

File: spec/helpers/entities_helpers_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe EntitiesHelper, :type => :helper do
+  describe "#active_string" do
```
Comment (josephbridgwaterrowe) [inline spec/helpers/entities_helpers_spec.rb]: My bad, thanks hound.

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review if this passes the build and hound.

Thanks.

**Comment by michaelachrisco** [comment]
Will do.

File: spec/helpers/entities_helpers_spec.rb (hunk 1 above)
Comment (michaelachrisco) [inline spec/helpers/entities_helpers_spec.rb]: Your most recent commit did not remove this issue.

Comment (josephbridgwaterrowe) [inline spec/helpers/entities_helpers_spec.rb]: Crap, I forgot to commit and push... it should be resolved now.

File: app/models/salesperson.rb [hunk 2]
```diff
@@ -21,17 +16,7 @@ class Salesperson < ActiveRecord::Base
...
-  end
-
-  def remove_location
-    default_location.destroy
-  end
-
-  def location=(location)
-    default_location = build_default_location if default_location.nil?
-    default_location.location = location
+    is_active == 1 ? true : false
```
Comment (michaelachrisco) [inline app/models/salesperson.rb]: I believe you can set this as:
```Rails
  def active?
    is_active == 1
```
Rails will default to true/false as result. No need for if/then.

Comment (josephbridgwaterrowe) [inline app/models/salesperson.rb]: Yes, I believe you are right.

**Pull request merged**